README.md

```diff
@@ -4,6 +4,25 @@ This Yocto meta layer contains all the recipes needed to build the Mender client
 
 For instructions on using this meta layer as part of a Yocto build environment, please see [the Mender documentation](https://docs.mender.io/system-updates-yocto-project).
 
+## Maintenance strategy
+
+The repository follows the release-named branch strategy.
+
+### Current branch state
+
+This branch `master` is in **development** maintenance.
+
+### Maintenance states
+
+A branch can be in one of the following maintenance states:
+
+| State | Maintenance level |
+| :---- | :---------------- |
+| bleeding edge | Expect breakage as we're working on things. No guarantees for functionality at any given point in time. |
+| development | Development once it has stabilized is applied here. Not recommended for production, breakage is considered a bug and will be fixed. Feature submissions should always be applied here first. |
+| stable | Ready for production. Stability takes precedence over features. |
+| end of life | No maintenance. Expect increasing security problems and breakage. Updating to a stable branch is highly recommended. |
+
```

Comment:
@TheYoctoJester 
This is just the groundwork terminology which will later be mapped to different branches?

Reply:
@TheMeaningfulEngineer exactly that.

Reply:
I don't get the `bleeding edge`.
The only thing I can think of as `bleeding edge` at the moment are the actual forked branches people work with, but that's to expensive to maintain in this readme.

How do you see `bleeding edge` used?

(Also @oleorhagen @lluiscampos )

Reply:
`bleeding edge` is our `master-next` for example, for where we don't run any CI tests nor nothing. Use at your own risk.

Reply:
`master-next` is the `bleeding edge` poster child. Might or might not work as people think it does.

Reply:
Makes sense, didn't know we had a `master-next`.
Thanks, resolveing.